public/app/features/dashboard/dashgrid/liveTimer.ts

```diff
@@ -69,11 +70,16 @@ class LiveTimer {
   measure = () => {
     const now = Date.now();
     this.budget = (now - this.lastUpdate) / interval;
-    this.ok = this.budget <= this.threshold;
+
+    const oldOk = this.ok.getValue();
+    const newOk = this.budget <= this.threshold;
+    if (oldOk !== newOk) {
+      this.ok.next(newOk);
+    }
     this.lastUpdate = now;
 
     // For live dashboards, listen to changes
-    if (this.ok && this.isLive && this.timeRange) {
+    if (this.ok.getValue() && this.isLive && this.timeRange) {
```

Comment:
```suggestion
    if (this.isLive && this.ok.getValue() && this.timeRange) {
```

maybe swap these so the cheaper `this.isLive` is first